pkg/parse/root.go

```diff
@@ -57,28 +57,30 @@ func NewRootRunner(clusterName, syncName, reconcilerName string, format filesyst
 		return nil, err
 	}
 
-	opts := opts{
-		clusterName:        clusterName,
-		syncName:           syncName,
-		reconcilerName:     reconcilerName,
-		client:             c,
-		pollingPeriod:      pollingPeriod,
-		resyncPeriod:       resyncPeriod,
-		retryPeriod:        retryPeriod,
-		statusUpdatePeriod: statusUpdatePeriod,
-		files:              files{FileSource: fs},
-		parser:             filesystem.NewParser(fileReader),
-		updater: updater{
-			scope:      declared.RootReconciler,
-			resources:  resources,
-			applier:    app,
-			remediator: rem,
+	return &root{
```

Comment:
This refactor just moves the `opts` construction into the `root` to avoid a copy of the mutex pointer, which fixes a linter error.